aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,195 @@
+
+## Choose a server rendered ASP.NET Core UI solution
+
+The following section briefly explains the ASP.NET Core web UI server rendered models available and directs next steps to get started.  ASP.NET Core Razor Pages and ASP.NET Core MVC are server-based frameworks for building web apps with .NET.
+
+### ASP.NET Core Razor Pages
+
+Razor Pages is a page-based model that is easy to get started with. UI and business logic concerns are kept separate, but within the page. Razor Pages is the preferred way to create page-based or form-based apps in ASP.NET Core.
+
+Razor Pages benefits, in addition to the server rendering benefits previously listed:
+
+* Quickly build and update UI.  What you need to code for the page is kept with the page, while keeping UI and business logic concerns separate.
+* Testable and scales to large apps.
+* Keep your ASP.NET Core pages organized in a simpler way:
+  * All view (page) specific logic and page properties (view model) can be kept together in their own namespace and directory.  
+  * Groups of related pages can be kept in their own namespace and directory.
+
+
+To get started with your first ASP.NET Core Razor Pages app, see [Get started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start). For a complete overview of ASP.NET Core Razor Pages, its architecture, and benefits,  [Introduction to Razor Pages in ASP.NET Core](xref:razor-pages/index)
+
+### ASP.NET MVC
+
+ASP.NET MVC renders UI on the server and uses a Model-View-Controller (MVC) architectural pattern. The MVC pattern separates an application into three main groups of components: Models, Views, and Controllers. User requests are routed to a Controller.  The Controller is responsible for working with the Model to perform user actions and, or retrieve results of queries. The Controller chooses the View to display to the user, and provides it with any Model data it requires.
+
+MVC benefits, in addition to the server rendering benefits previously listed:
+
+* Easier to scale the web UI application in terms of complexity. It's easier to code, debug, and test something (model, view, or controller) that has a single job.
+* Clear [separation of concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns) for maximum flexibility.
+* The Model-View-Controller separation of responsibilities ensures that the business model can evolve without being tightly coupled to low-level implementation details.
+
+To get started with your first ASP.NET Core MVC app, see [Get started with MVC](xref:tutorials/first-mvc-app/start-mvc). For a complete overview of ASP.NET Core MVC, its architecture and benefits, see [Overview of ASP.NET Core MVC](xref:mvc/overview)
+
+## Choose a client rendered ASP.NET Core solution
+
+The following section briefly explains the ASP.NET Core web UI client rendered models available and directs you to the next steps to get started.
+
+### ASP.NET Core Blazor WebAssembly
```

Comment:
```suggestion
### Blazor
```

We have content describing the tradeoffs with the different Blazor hosting models here: https://docs.microsoft.com/aspnet/core/blazor/hosting-models. I'd like to avoid duplicating that content here. Instead I think this section should focus on Blazor as a client-side web UI framework (independent of hosting model) and the tradeoffs of using Blazor vs other JS frameworks.